### ISSUE TITLE
TypeError when setting up the database 

### ISSUE THREAD

**Issue by Jeroendevr** [Kyria/LazyBlacksmith#92]
When going through the process of installing en setting up the application I come to the chapter where I should run 
`python manage.py db upgrade` 
according to 
[Getting started](https://kyria.github.io/LazyBlacksmith/getting_started/installation/)

```
LazyBlacksmith % python manage.py db upgrade
Traceback (most recent call last):
  File "/Users/jdv/Programming/LazyBlacksmith/manage.py", line 6, in <module>
    from lazyblacksmith.app import create_app
  File "/Users/jdv/Programming/LazyBlacksmith/lazyblacksmith/app.py", line 14, in <module>
    from lazyblacksmith.utils.context_processor import inject_user
  File "/Users/jdv/Programming/LazyBlacksmith/lazyblacksmith/utils/context_processor.py", line 3, in <module>
    import lazyblacksmith.models.enums as enums
  File "/Users/jdv/Programming/LazyBlacksmith/lazyblacksmith/models/__init__.py", line 22, in <module>
    from .sde.region import Region
  File "/Users/jdv/Programming/LazyBlacksmith/lazyblacksmith/models/sde/region.py", line 5, in <module>
    class Region(db.Model):
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/flask_sqlalchemy/model.py", line 67, in __init__
    super(NameMetaMixin, cls).__init__(name, bases, d)
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/flask_sqlalchemy/model.py", line 121, in __init__
    super(BindMetaMixin, cls).__init__(name, bases, d)
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/sqlalchemy/orm/decl_api.py", line 75, in __init__
    _as_declarative(reg, cls, dict_)
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/sqlalchemy/orm/decl_base.py", line 126, in _as_declarative
    return _MapperConfig.setup_mapping(registry, cls, dict_, None, {})
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/sqlalchemy/orm/decl_base.py", line 177, in setup_mapping
    return cfg_cls(registry, cls_, dict_, table, mapper_kw)
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/sqlalchemy/orm/decl_base.py", line 314, in __init__
    self._early_mapping(mapper_kw)
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/sqlalchemy/orm/decl_base.py", line 200, in _early_mapping
    self.map(mapper_kw)
  File "/Users/jdv/Programming/LazyBlacksmith/env/lib/python3.9/site-packages/sqlalchemy/orm/decl_base.py", line 977, in map
    mapper_cls(self.cls, self.local_table, **self.mapper_args),
TypeError: __init__() got an unexpected keyword argument 'order_by'
```

Perhaps the following is relevant [other sqlalchemy issue on GitHub](https://github.com/marshmallow-code/marshmallow-sqlalchemy/issues/235)

**Comment by BtbN** [comment]
I fixed that a bunch of other issues with newer sqlalchemy in https://github.com/Kyria/LazyBlacksmith/pull/91

**Comment by Kyria** [comment]
this is fixed in the 1.0.5 release (at least when I just tested with it, it didn't happen) 

feel free to open again the issue if it's not solved :)

Thanks!
